helm/chaos-mesh/values.yaml

```diff
@@ -139,6 +139,13 @@ controllerManager:
     retryPeriod: 2s
   # chaosdSecurityMode is enabled for mTLS connection between chaos-controller-manager and chaosd
   chaosdSecurityMode: true
+  # multi cluster install offline helm chart path
+  localHelmChart:
+    enabled: true
+    volume:
+      hostPath:
+        path: /data/helm
+        type: DirectoryOrCreate
```

Comment:
The default value for `controllerManager.localHelmChart.enabled` is `true`, and the default value of `controllerManager.localHelmChart.volume` is a `HostPath` which most probably does NOT exist on the environment. It would lead to the installation failing with the default values.

What about:

- making `controllerManager.localHelmChart.enabled`  as `false`
- also make the `controllerManager.localHelmChart.volume` as a volume for ConfigMap or Secret, with some comments.